crates/core/src/live_socket/mod.rs

```diff
@@ -69,6 +66,16 @@ impl Default for UploadConfig {
         }
     }
 }
+#[derive(uniffi::Object)]
+pub struct LiveChannel {
+    pub channel: Arc<Channel>,
+    pub socket: Arc<Socket>,
+    pub join_payload: Payload,
+    document: FFiDocument,
+    dead_render: FFiDocument,
+    style_urls: Vec<String>,
```

Comment:
I think these should only be on the `LiveSocket`, since we just read them once after the socket connects, not for each channel that's joined.

Reply:
Yeah, I was also wondering about that. I'll expose it only in `LiveSocket`.